source/extensions/filters/http/grpc_stats/config.cc

```diff
@@ -0,0 +1,135 @@
+#include "extensions/filters/http/grpc_stats/config.h"
+
+#include "envoy/http/filter.h"
+#include "envoy/registry/registry.h"
+
+#include "common/grpc/common.h"
+#include "common/grpc/context_impl.h"
+
+#include "extensions/filters/http/grpc_stats/message_counter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace GrpcStats {
+
+namespace {
+
+class GrpcStatsFilter : public Http::StreamFilter {
```

Comment:
nit: can we split the actual filter out into a different h/cc file or not call this file config.cc